docs/rules_pkl_docs.md

```diff
@@ -0,0 +1,171 @@
+
+## pkl_toolchain
+
+<pre>
+pkl_toolchain(<a href="#pkl_toolchain-name">name</a>, <a href="#pkl_toolchain-cli">cli</a>)
+</pre>
+
+
+
+**ATTRIBUTES**
+
+
+| Name  | Description | Type | Mandatory | Default |
+| :------------- | :------------- | :------------- | :------------- | :------------- |
+| <a id="pkl_toolchain-name"></a>name |  A unique name for this target.   | <a href="https://bazel.build/concepts/labels#target-names">Name</a> | required |  |
+| <a id="pkl_toolchain-cli"></a>cli |  -   | <a href="https://bazel.build/concepts/labels">Label</a> | optional |  `None`  |
+
+
+<a id="pkl_doc"></a>
+
+## pkl_doc
+
+<pre>
+pkl_doc(<a href="#pkl_doc-name">name</a>, <a href="#pkl_doc-srcs">srcs</a>, <a href="#pkl_doc-kwargs">kwargs</a>)
+</pre>
+
+Generate documentation website for Pkl files.
+
+**PARAMETERS**
+
+
+| Name  | Description | Default Value |
+| :------------- | :------------- | :------------- |
+| <a id="pkl_doc-name"></a>name |  A unique name for this target.   |  none |
+| <a id="pkl_doc-srcs"></a>srcs |  The Pkl source files to be documented.   |  none |
+| <a id="pkl_doc-kwargs"></a>kwargs |  Further keyword arguments. E.g. visibility   |  none |
+
+
+<a id="pkl_test_suite"></a>
+
+## pkl_test_suite
+
+<pre>
+pkl_test_suite(<a href="#pkl_test_suite-name">name</a>, <a href="#pkl_test_suite-srcs">srcs</a>, <a href="#pkl_test_suite-deps">deps</a>, <a href="#pkl_test_suite-tags">tags</a>, <a href="#pkl_test_suite-visibility">visibility</a>, <a href="#pkl_test_suite-size">size</a>, <a href="#pkl_test_suite-test_suffix">test_suffix</a>, <a href="#pkl_test_suite-kwargs">kwargs</a>)
+</pre>
+
+Create a suite of Pkl tests from the provided files.
+
+Given the list of `srcs`, this macro will generate:
+
+1. A `pkl_test` target (with visibility:private) per `src` that ends with `test_suffix`
+2. A `pkl_library` that accumulates any files that don't match `test_suffix`
+3. A `native.test_suite` that accumulates all of the `pkl_test` targets
+
+
+**PARAMETERS**
+
+
+| Name  | Description | Default Value |
+| :------------- | :------------- | :------------- |
+| <a id="pkl_test_suite-name"></a>name |  A unique name for this target.   |  none |
+| <a id="pkl_test_suite-srcs"></a>srcs |  The source files, containing Pkl test files. Accepts files with the .pkl extension.   |  none |
+| <a id="pkl_test_suite-deps"></a>deps |  Other targets to include in the Pkl module path when building this configuration. Must be pkl_* targets.   |  `None` |
+| <a id="pkl_test_suite-tags"></a>tags |  Tags to add to each Pkl test target.   |  `[]` |
+| <a id="pkl_test_suite-visibility"></a>visibility |  The visibility of non test Pkl source files.   |  `None` |
+| <a id="pkl_test_suite-size"></a>size |  Size of Pkl test.   |  `None` |
+| <a id="pkl_test_suite-test_suffix"></a>test_suffix |  A custom suffix indicating a source file is a Pkl test file.   |  `None` |
+| <a id="pkl_test_suite-kwargs"></a>kwargs |  Further keyword arguments.   |  none |
+
+
```

Comment:
Any plans on making this an Antora module that we can add to pkl-lang.org?

Reply:
is this something we can do as a follow-up PR?